packages/SwingSet/src/kernel/virtualObjectManager.js

```diff
@@ -188,6 +193,61 @@ export function makeVirtualObjectManager(
     syscall.vatstoreSet(`vom.${vobjID}`, JSON.stringify(rawData));
   }
 
+  function possibleVirtualObjectDeath(vobjID) {
+    if (!isVrefReachable(vobjID) && !getValForSlot(vobjID)) {
+      const [exported, refCount] = getRefCounts(vobjID);
+      if (exported === 0 && refCount === 0) {
+        syscall.vatstoreDelete(`vom.${vobjID}`);
+        syscall.vatstoreDelete(`vom.${vobjID}.refCount`);
+      }
```

Comment:
Let's add a comment here like "todo: decrement vrefs in the virtualized data being deleted", to remind ourselves that this is where the next phase will get implemented.